edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -80,11 +97,54 @@ def authenticate(self, request):
             # Errors in production do not need to be logged (as they may be noisy),
             # but debug logging can help quickly resolve issues during development.
             logger.debug('Failed JWT Authentication,', exc_info=exception)
-            # Note: I think this case should only include AuthenticationFailed and PermissionDenied,
-            #   but will monitor the custom attribute to verify.
+            # Useful monitoring for debugging various types of failures.
             set_custom_attribute('jwt_auth_failed', 'Exception:{}'.format(repr(exception)))
             raise
 
+    def _authenticate_forgiving_jwt_cookies(self, request):
+        """
+        Authenticate using a JWT token.
+
+        If the JWT is in the authorization header, and authentication fails, we will raise
+            an exception which will halt additional authentication methods, and is the default
+            behavior of DRF authentication classes.
```

Comment:
```suggestion
        If the JWT is in the authorization header, and authentication fails, we will raise
        an exception which will halt additional authentication methods, and is the default
        behavior of DRF authentication classes.
```

Reply:
Curious if we have a style documentation around this? I think I was introduced to the indenting by Régis, but could have been someone else, and I prefer the indents. It means multi-paragraph comments don't need blank lines, which clarifies that they are grouped. Non-blocking either way, but just curious if this is simply competing preferences.